### PR TITLE
Finish css

### PR DESCRIPTION
**Pull request by CorentinCls**

### CODE REVIEW

File: src/View/Components/_hashtag.html.twig [hunk 1]
```diff
@@ -1,7 +1,7 @@
-<section class="listHashtag">
+<section class="listHashtag list-hashtag-create">
     <ul>
         <li>#Murder</li>
-        <li>#LondresCrime</li>
+        <li>#LondreCrime</li>
```
Comment (MarjorieThomassin) [inline src/View/Components/_hashtag.html.twig:4]: there is a "s" at the end of the word "Londres"

File: src/View/Crime/createCrime.html.twig [hunk 2]
```diff
@@ -1,8 +1,12 @@
 {% extends 'layout.html.twig' %}
 
 {% block content %}
-    {% include 'Components/_header.html.twig' %}
-    {% include 'Components/_hashtag.html.twig' %}
-    {% include 'Components/_formCrime.html.twig' %}
+    <div class="main">
+        {% include 'Components/_header.html.twig' %}
+        <div>
```
Comment (MarjorieThomassin) [inline src/View/Crime/createCrime.html.twig:6]: no class on this div, is it really useful ?

**Review by MarjorieThomassin — CHANGES_REQUESTED** [review]

File: public/assets/css/style.css [hunk 3]
```diff
@@ -120,13 +124,10 @@ input::placeholder {
         background: none;
         border: none;
         outline: none;
-        padding-bottom: 55px;
-}
-      
-#searchQuerySubmit:hover {
+        padding-bottom: 10px;
```
Comment (MarjorieThomassin) [inline public/assets/css/style.css:130]: replace px by rem

File: public/assets/css/style.css [hunk 4]
```diff
@@ -290,12 +267,22 @@ input::placeholder {
 }
 
 .link-home {
+        margin: auto;
         text-decoration: none;
-        color: var(--secondary-color);
+        background: #c2bcaa;
+        color: rgb(0, 0, 0);
+        border-radius: 4px;
+        padding: 5px;
```
Comment (MarjorieThomassin) [inline public/assets/css/style.css:286]: same here, padding and margin always in rem :)

File: public/assets/css/style.css [hunk 5]
```diff
@@ -290,12 +267,22 @@ input::placeholder {
...
+        width: 60%;
+        margin-top: 5px;;
```
Comment (MarjorieThomassin) [inline public/assets/css/style.css:288]: same here

**Review by MarjorieThomassin — CHANGES_REQUESTED** [review]

File: public/assets/css/style.css [hunk 6]
```diff
@@ -394,8 +398,10 @@ input::placeholder {
         text-decoration: none;
         background: #c2bcaa;
         color: rgb(0, 0, 0);
-        border-radius: 10px;
-        padding: 0.5rem;
+        border-radius: 4px;
+        padding: 5px;
```
Comment (MarjorieThomassin) [inline public/assets/css/style.css:420]: rem for padding and margin

**Review by MarjorieThomassin — CHANGES_REQUESTED** [review]

File: src/View/Crime/createCrime.html.twig (hunk 2 above)
Comment (CorentinCls) [reply src/View/Crime/createCrime.html.twig:6]: ? I use a class called main

**Pull request merged**